front/app/components/Form/Components/Layouts/CLPageLayout.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { memo, useState, useEffect, useContext } from 'react';
+import { LayoutProps, RankedTester, rankWith } from '@jsonforms/core';
+import { JsonFormsDispatch, withJsonFormsLayoutProps } from '@jsonforms/react';
+import styled, { useTheme } from 'styled-components';
+import Ajv from 'ajv';
+
+// Components
+import {
+  Box,
+  Title,
+  useBreakpoint,
+  media,
+} from '@citizenlab/cl2-component-library';
+import { FormSection } from 'components/UI/FormComponents';
+import Button from 'components/UI/Button';
+
+// Context
+import { FormContext } from 'components/Form/contexts';
+
+// i18n
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../../messages';
+
+// Utils
+import {
+  getSanitizedFormData,
+  getPageSchema,
+  PageCategorization,
+  isPageCategorization,
+} from 'components/Form/Components/Layouts/utils';
+
+const StyledFormSection = styled(FormSection)`
+  max-width: 100%;
+  width: 100%;
+
+  ${media.phone`
+    padding-left: 18px;
+    padding-right: 18px;
+  `}
+`;
```

Comment:
Could we apply this styling on a Box wrapper for the FormSection instead? To avoid using styled components?

Reply:
`FormSection` is used uniformly within the project. I didn't want to make such a change within this PR. Seems out of scope.

Reply:
Ah sorry - I was thinking more of just wrapping the FormSection within this component, like we've discussed at the FE meetings before:
E.g.
```
<Box maxWidth="100%" width="100%" px={isMobile ? "18px" : "0px (or whatever the default it)"}>
      <FormSection/>
</Box>
```

Reply:
Doing this will not work because it won't override the inherent styles that we want to override here. The wrapping works if want to add certain styles.